src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandomDesktop.java

```diff
@@ -0,0 +1,151 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsFandomContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoFanTakeover;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(
+        groups = "AdsVuapFandomDesktop"
+)
+public class TestAdsVuapFandomDesktop extends AdsFandomTestTemplate{
+    private static final String URL_FIRSTQUARTILE = "ad_vast_point=firstquartile";
+    private static final String URL_MIDPOINT = "ad_vast_point=midpoint";
+    private static final int DELAY = 2;
+    private static final int VIDEO_START_TIME = 0;
+
+    @Test(
+            dataProviderClass = FandomAdsDataProvider.class,
+            dataProvider = "fandomVuapPage",
+            groups = "AdsVideoClosedAfterPlayingFandom"
+    )
+    public void adsVideoClosedAfterPlayingFandom(String pageType, String pageName, String slotName, String iframeId) {
+        AdsFandomObject fandomPage = loadPage(pageName, pageType);
+        fandomPage.triggerOnScrollSlots();
+        VideoFanTakeover videoFanTakeover = new VideoFanTakeover(driver, iframeId);
+        fandomPage.scrollToSlot(AdsFandomContent.getSlotSelector(slotName));
```

Comment:
What do you think about extracting those four lines above in separate method and call it prepare/init or sthg like that? I can see that this is executed in all tests in this file. What do you think?